tasks/compile-example.js

```diff
@@ -26,13 +26,18 @@ const DocumentParser = require('../lib/DocumentParser');
 const ExampleFile = require('../lib/ExampleFile');
 const Metadata = require('../lib/Metadata');
 const Templates = require('../lib/Templates');
-const storyController = fs.readFileSync(__dirname + '/../templates/stories/page-switch.html', 'utf8');
+const storyController =
+  fs.readFileSync(__dirname + '/../templates/stories/page-switch.html', 'utf8');
 const storyBookend = require('./bookend.json');
 
 const STORY_EMBED_DIR = __dirname + '/../api/dist/';
-const AMP_STORY_CLEANER_REGEX = ['amp-story', 'amp-story-auto-ads', 'amp-consent'].map(extension =>
-  new RegExp('<script\\s+async\\s+custom-element="' + extension + '"\\s+src="https:\\/\\/cdn\\.ampproject\\.org\\/v0\\/' + extension + '-\\d\\.\\d\\.js"><\\/script>')
-);
+const AMP_STORY_CLEANER_REGEX =
+  ['amp-story', 'amp-story-auto-ads', 'amp-consent']
+      .map(extension =>
+        new RegExp('<script\\s+async\\s+custom-element="' + extension
```

Comment:
Nit: I recommend using backticks here, that was we avoid all those `\\` and it's also easier to concatenate the `extension` variable.

Reply:
* the `\\` are needed when you init a regex with a string (instead of `/.../`). 
* backticks have no real advantage here as I need to break the lines anyway because of eslint...